client/devdocs/design/index.jsx

```diff
@@ -125,3 +137,33 @@ export default React.createClass( {
 		);
 	}
 } );
+
+DesignAssets.propTypes = {
```

Comment:
Actually I was thinking to add them only when the feature is enabled. Does it make sense?